pegasus/sites.v3/code.org/views/frozen_languages.haml

```diff
@@ -0,0 +1,6 @@
+%h2 
+  %a{href: CDO.studio_url('/s/frozen')} The Frozen tutorial is supported in the following languages
+%ul
+  - Languages.get_frozen_languages.each do |properties|
+    -if properties[:frozen_b] == true
```

Comment:
You can also simplify this Ruby code. Instead of `- if properties[:frozen_b] == true` just do `- if properties[:frozen_b]`.